app/Actions/DisplayCurrentConfiguration.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Actions;
+
+use function is_bool;
+use function is_string;
+use App\Support\BaseAction;
+
+class DisplayCurrentConfiguration extends BaseAction
+{
+    /**
+     * Displays current configuration.
+     *
+     * @return void
+     */
+    public function __invoke(): void
+    {
+        $rows = config('lambo', []);
+
+        $rows = collect($rows)->filter(function ($item, $key) {
+            return $key !== 'after';
```

Comment:
this can become a `reject()` method instead of `filter()` and you can drop the `!`